karma.conf.js

```diff
@@ -78,7 +78,6 @@ module.exports = (config) => {
             preserveDescribeNesting: true,
             foldAll: false
         },
-        browserConsoleLogOptions: { level: "error",  format: "%b %T: %m",  terminal: true },
```

Comment:
do you mind if we leave this in? I think it could be useful in the future

Reply:
hm, actually it was suppressing logs and hiding noise.
I kinda cleanup up all noise so useful errors/warning/logs can be more visible and not hidden in noise.

Reply:
Does the `format` mean it only suppresses the deprecation warnings? Because that's fine.

Reply:
It's suppressing any logs.